guides/_posts/2014-01-5-vfl.html

```diff
@@ -22,26 +34,26 @@
 #panel[right] - 10 == #button4[right];
 {% endhighlight %}
 
-Not to mention the obligatory size & vertical constraints needed to fully specify the panel and buttons - ugh...  
+Not to mention the obligatory size & vertical constraints needed to fully specify the panel and buttons - ugh...
 
 This is where Grid-flavored VFL comes in.  The brains at Apple came up with VFL as a more programatic way to install constraints in Cocoa AutoLayout.  The idea is rather simple, the syntax should visually depict the layout.  GSS takes the good parts of VFL and makes it more suitable for CSS, throws in some sugar and drops the cruft.  So the above example of 4 buttons in a panel becomes:
 
 {% highlight css %}
-@horizontal |-[#button1]-[#button2]-[#button3]-[#button4]-| in(#panel) gap(10);
+@horizontal |-(#button1)-(#button2)-(#button3)-(#button4)-| in(#panel) gap(10);
 
 /* which is the same as */
 
-@horizontal |-10-[#button1]-10-[#button2]-10-[#button3]-10-[#button4]-10-| in(#panel);
+@horizontal |-10-(#button1)-10-(#button2)-10-(#button3)-10-(#button4)-10-| in(#panel);
```

Comment:
I personally think we should promote the use of `@h` and `@v` exclusively and remove other ways from the CCSS parser. Would be good to get some other people's input.